### DISCUSSION TITLE
Vim python mode

### DISCUSSION THREAD

**Discussion by klieret** [JetBrains/ideavim#535] [Ideas]
[$VIMRUNTIME/ftplugin/python.vim](https://github.com/vim/vim/blob/master/runtime/ftplugin/python.vim)  remaps all builtin mappings documented under `:h ]]` and `:h ]m` for the python language. 

However, this does not seem to be active in pycharm with ideavim.

Alternatively, the `python-mode` plugin does something similar.

More background in [this SO post](https://stackoverflow.com/a/28284564/)

**Comment by klieret** [comment]
Similarly, `match.it` is useless in python/pycharm without configuration for python: https://www.vim.org/scripts/script.php?script_id=386

**Comment by joaomcarlos** [comment]
@klieret did you end up getting "]m" and similar working with PyCharm? How?

**Reply by volard** [reply]
I've done this in `.ideavimrc`:
```
nmap ]m :action MethodDown<CR>
nmap [m :action MethodUp<CR>
```
